Source/WebKit/WebProcess/WebPage/RemoteLayerTree/RemoteLayerTreeDrawingArea.mm

```diff
@@ -307,6 +307,9 @@
         return;
     }
 
+    if (m_inDisplayDidRefresh)
```

Comment:
Rather than maintaining `m_inDisplayDidRefresh`, which is fragile, can we just pass an argument in that tells us if this came out of the display link call chain? The "other" call sites could pass a non-default value for the argument.

Reply:
I can, it's just a somewhat complicated call chain from the `RenderingUpdateScheduler` in WebCore, through to all the different `DrawingArea` implementations.